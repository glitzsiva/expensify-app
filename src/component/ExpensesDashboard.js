import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesListFilters from './ExpensesListFilters';
import ExpensesSummary from './ExpensesSummary';



const ExpenseDashboard=()=>(
    <div> 
        Dasboard page
        <ExpensesSummary />
        <ExpensesListFilters />
        <ExpenseList />
        
    </div>
);


export default ExpenseDashboard;