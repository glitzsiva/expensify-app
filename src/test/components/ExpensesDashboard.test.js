import React from 'react';
import {shallow} from 'enzyme';
import ExpensesDashboard from '../../component/ExpensesDashboard';

test('expenses Dasboard ',()=>{
    const wrapper =shallow(<ExpensesDashboard />);
    expect(wrapper).toMatchSnapshot();
})