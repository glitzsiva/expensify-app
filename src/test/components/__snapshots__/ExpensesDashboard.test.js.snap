// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expenses Dasboard  1`] = `
<div>
  Dasboard page
  <Connect(ExpensesSummary) />
  <Connect(ExpensesListFilters) />
  <Connect(ExpenseList) />
</div>
`;
